spec/models/concerns/publishing_api/metadata_spec.rb

```diff
@@ -173,7 +173,21 @@
           }
         end
 
-        it { is_expected.to match_array(%w[0000 1111 2222 3333 4444 5555]) }
+        it { is_expected.to eq(%w[4444 5555 0000 1111 2222 3333]) }
```

Comment:
It does read a bit strangely, but this really helps illustrate what the code is doing. Thanks!